manifests/0000_90_console-operator_01_prometheusrbac.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Console
```

Comment:
Should these be lowercase `console`?

Reply:
no, we're pivoting to PascalCase for the enum values